include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -17,6 +17,8 @@
 #ifndef GLOW_IMPORTER_COMMONOPERATORLOADER_H
 #define GLOW_IMPORTER_COMMONOPERATORLOADER_H
 
+#include "onnx/onnxifi.h"
```

Comment:
It seems like CommonOperatorLoader shouldn't need to know about ONNX or ONNXIFI.  Is there a way to avoid that?

Reply:
Hmm, one thing that is going to be in common is the ability to load weights from onnxTensorDescriptor. For that we need onnxifi.h.

Reply:
Got it.  That seems fine then.